### ISSUE TITLE
Illegal instruction core dumped error

### ISSUE THREAD

**Issue by Remimstr** [Khezen/docker-mongo-rocks#15]
Hi, I'm loving this image so far; I think it's really well built and easy to set up :)

However, I did run into a problem when trying to use this image on our linux server instead of my local mac machine. Whenever I try to run any version of the image, it either fails with a `13 Illegal Instruction (core dumped) $cmd` error, hangs on "=> Waiting for confirmation of MongoDB service startup", or does both. I know the error has something to do with mongod because when I try to run mongod from within a container (bypassing the entrypoint), I get the same error.

I'm going to continue investigating this problem and hopefully solve it within a couple of days, in which case I'll issue a PR if appropriate. In the meantime, any thoughts you have would be very welcome :)

Kernel: 3.16.0-4-amd64
Linux Version: 
- Distributor ID:	Debian
- Description:	Debian GNU/Linux 8.8 (jessie)
- Release:	8.8
- Codename:	jessie

Also, I have had no problems running a previous (non-sharded) instance of mongo on this server within a docker container.

**Comment by Remimstr** [comment]
Fixed via #16

**Comment by Remimstr** [comment]
Hi Khezen,

I've continued to investigate this issue over the past couple of days and haven't come up with much to be honest. I realize that the way I changed the image to install mongo is incompatible with the rocksdb implementation and that your approach is the correct one. On that note however, I simply cannot get your image to work on our server. I tried building the image right on our server, and once I got past the "not enough free space errors", received this as a result while running scons on the mongo-rocks repo:
 
![screenshot](https://user-images.githubusercontent.com/11297614/28465738-d106f55c-6df7-11e7-9a3c-6801832c5d15.png)

I then attempted to replicate the issue with a new build and succeeded in building the image. When I went to run it though, I got an entirely new error: 

`mongod: error while loading shared libraries: librocksdb.so.5.4: cannot open shared object file: No such file or directory`

At this point, I'm leaning towards using my fork that excludes rocksdb because it's simpler, smaller and seemingly more stable. That being said, I am open to helping debug this if we want to continue this investigation.
